ext/standard/crypt_sha256.c

```diff
@@ -443,7 +446,7 @@ char * php_sha256_crypt_r(const char *key, const char *salt, char *buffer, int b
 	sha256_finish_ctx(&alt_ctx, temp_result);
 
 	/* Create byte sequence P.  */
-	cp = p_bytes = alloca(key_len);
+	cp = p_bytes = do_alloca(key_len, use_heap_key);
```

Comment:
I think these would also need separate use_heap variables?

Reply:
I reused it because it if it is used previously the size is ``key_len + __alignof__(uint32_t)`` which is *nearly* the same, but I can use another variable.